modules/app-gateway-module/config.tf

```diff
@@ -0,0 +1,47 @@
+locals {
+  vnet_rg   = "aks-infra-${var.env}-rg"
+  vnet_name = "core-${var.env}-vnet"
+
+  criticality = {
```

Comment:
same for criticality. I think we should do it via the component rather than the module. It will be easy if someone wants to re-use the module for some other project or environment